### PR TITLE
Add support for custom help screens

### PR DESCRIPTION
**Pull request by kodie**
This allows us to set a custom help screen for either Applications or Commands.

```php
$app = new \Ahc\Cli\Application('Test App', '0.0.1');
$app->help('My application\'s custom help screen');

$cmd = $app->command('cmd');
$cmd->help('My command\'s custom help screen');
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,11 +170,16 @@ class InitCommand extends Ahc\Cli\Input\Command
     {
         parent::__construct('init', 'Init something');
 
+        $help = '<cyan>Custom help screen</end>';
+        $writer = new Ahc\Cli\Output\Writer();
+        $rendered_help = $writer->colorizer()->colors($help);
```
Comment (adhocore) [inline README.md:175]: $camelCase or it can be just done below: `->help($writer->colorizer()->colors($help))`

File: src/Application.php [hunk 2]
```diff
@@ -52,6 +52,10 @@ class Application
     /** @var string Ascii art logo */
     protected string $logo = '';
 
+    /** @var string Custom help screen */
+    protected string $custom_help = '';
```
Comment (adhocore) [inline src/Application.php:56]: `$help` is good enough name i think

File: README.md (hunk 1 above)
Comment (kodie) [reply README.md]: Updated!

File: src/Application.php (hunk 2 above)
Comment (kodie) [reply src/Application.php]: Done!

**Review by adhocore — APPROVED** [review]

**Pull request merged**